include/TestProb.h

```diff
@@ -40,8 +40,9 @@ extern double (*Mis_GetTimeStep_User_Ptr)( const int lv, const double dTime_dt )
 extern void (*Mis_UserWorkBeforeNextLevel_Ptr)( const int lv, const double TimeNew, const double TimeOld, const double dt );
 extern void (*Mis_UserWorkBeforeNextSubstep_Ptr)( const int lv, const double TimeNew, const double TimeOld, const double dt );
 extern void (*Aux_Record_User_Ptr)();
-extern void (*BC_User_Ptr)( real fluid[], const double x, const double y, const double z, const double Time,
-                            const int lv, double AuxArray[] );
+extern void (*BC_User_Ptr)( real Array[], const int ArraySize[], real fluid[], const int NVar_Flu,
```

Comment:
Probably because of this change, there is an error at `TestProblem/Hydro/Bondi/Init_TestProb_Hydro_Bondi.cpp(636)`: 
```
 error: a value of type "void (*)(real={float} *, double, double, double, dou
ble, int, double *)" cannot be assigned to an entity of type "void (*)(real={float} *, const int *, real={float} *, int, int, const int 
*, const double *, double, int, const int *, double *)"
     BC_User_Ptr              = SetGridIC;
                              ^

```
when compiling with `--model=HYDRO --fftw=FFTW3 --gravity=true`